Content.Server/Blob/BlobMobComponent.cs

```diff
@@ -0,0 +1,22 @@
+using Content.Shared.Damage;
+using Content.Shared.FixedPoint;
+
+namespace Content.Server.Blob;
+
+[RegisterComponent]
+public sealed class BlobMobComponent : Component
+{
+    [ViewVariables(VVAccess.ReadOnly), DataField("healthOfPulse")]
+    public DamageSpecifier HealthOfPulse = new()
+    {
+        DamageDict = new Dictionary<string, FixedPoint2>
+        {
+            { "Blunt", -4 },
+            { "Slash", -4 },
+            { "Piercing", -4 },
+            { "Heat", -4 },
+            { "Cold", -4 },
+            { "Shock", -4 },
+        }
+    };
```

Comment:
Do not hardcode this in here, specify it in the prototype. Make this a required field.